Bottom: 5320ac4de600b00402745e4464e48ee4585352f2
Top:    5a85dbcc4aeda09b546416c49294ff9a3ce63fdd
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-21 12:13:12 +0800

Refresh of autofs-5.0.5-add-master-read-wait-option.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index d9d4569..05196f4 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1944,6 +1944,7 @@ int main(int argc, char *argv[])
 	unsigned ghost, logging, daemon_check;
 	unsigned dumpmaps, foreground, have_global_options;
 	unsigned master_read, master_wait;
+	int master_wait;
 	time_t timeout;
 	time_t age = time(NULL);
 	struct rlimit rlim;
